### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by bansp** [clarin-eric/standards#317]
I have taken a glimpse at the[ slides on web accessibility in CLARIN](https://www.clarin.eu/sites/default/files/Tone%20Merete%20Bruvik-TCC-meeting-Accessibility.pdf). The SIS is explicitly mentioned there with a score of 70%, which is not entirely bad, given that the best CLARIN site scores at 87%, while the worst at 50%. But we can do better and this ticket is designed to gather issues that target accessibility.

First, a list of the most important issues. I have numbered them for ease of referencing.

1. Use good contrast.
2. Black and white is not bad.
3. Does it work on phones and laptops?
4. Give alternative text to graphics and pictures.
5. Underline your links and state the purpose in the text of the link.
7. State the languages in use.
8. Encode your structures.
  a. Use regions like header, main and footer.
  b. Use headlines hierarchies.

There are tools that assist in checking sites for the above properties and others. In my Chromium, I can see [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) (under dev tools), but I don't yet understand its output. Another free tool mentioned in the slides is [WAVE](https://wave.webaim.org/).

One thought that occurs to me is that the aesthetics of the SIS is not fully in line with the above hints (compare \#5), but we would probably prefer to keep it, at least as a default. So the first question is: how sensible would it be to install a switch from the default SIS interface to one with enhanced accessibility features? I guess I should turn that into a separate issue...

**Comment by bansp** [comment]
To be sure, we can work on certain features (such as ensuring metainformation in the generated HTML `<head>` sections) without a switch. The switch would work for, e.g., link decoration, colour pallette, etc. -- the visible elements.

**Comment by bansp** [comment]
https://wave.webaim.org/report#/https://standards.clarin.eu/sis/index.xq
-- that is pretty cool. For example, it shows that we should add `lang="en"` to the `<html>` element -- that's something to be fixed right away. But then, it points to low contrast between the text and the background -- and that _is_ a matter of a switch, I'd say. 
We should go through them one by one (but first hunt for examples and discussion of an auto-triggered (?) switch; maybe it's hiding somewhere in the CSS media selectors?).

**Comment by bansp** [comment]
This is the initial state, produced with Lightning (emulating desktop, analysing a snapshot of the home page), for the record:
![image](https://github.com/user-attachments/assets/c766ba75-a0f5-4592-8255-e8a74a224bff)

I have submitted one fix, but the rest of the score is going to take quite a while, I'm afraid.
